neo/Implementations/Wallets/peewee/UserWallet.py

```diff
@@ -102,18 +126,23 @@ def OnCreateAccount(self, account):
 
         db_account, created = Account.get_or_create(
             PrivateKeyEncrypted=encrypted_pk, PublicKeyHash=account.PublicKeyHash.ToBytes())
-        db_account.PrivateKeyEncrypted = encrypted_pk
+        if not created:
```

Comment:
Could you please explain the change here? Not quite clear to me what is the advantage now.

Reply:
My initial change looked like
```
        db_account, created = Account.get_or_create(
            PrivateKeyEncrypted=encrypted_pk, PublicKeyHash=account.PublicKeyHash.ToBytes())
        if not created:
            db_account.PrivateKeyEncrypted = encrypted_pk
            db_account.save()
```
but then I realised that `Account.get_or_create` does not automatically save and that I had to change the indentation on `db_account.save()`. 

Now when thinking about it I believe 
```
        if not created:
            db_account.PrivateKeyEncrypted = encrypted_pk
```
should actually be removed completely, because it is always redundant. `PrivateKeyEncrypted` and `PublicKeyHash` are supplied as [filters](http://docs.peewee-orm.com/en/latest/peewee/api.html#Model.get_or_create). If it finds a record that satisfies them, then assigning the `encrypted_pk` is redundant. If no record exist that matches the filters, then it will use the already supplied filter/arguments to create a new record.

Reply:
Thanks, great explanation. Yeah, this line should probably just be removed.

Reply:
will do